habitat-aspnet-sample/plan.sh

```diff
@@ -0,0 +1,49 @@
+pkg_name=habitat-aspnet-sample
+pkg_origin=core
+pkg_version=0.1.9
+pkg_source=https://github.com/mwrock/${pkg_name}/archive/v${pkg_version}.tar.gz
```

Comment:
Is there any reason we're downloading this from your GitHub repo instead of just generating it with `dotnet new`?